src/components/navigation/Navbar.scss

```
@@ -0,0 +1,148 @@
+@import '../../constants.scss';
+
+.emptySpacingBox{
+  width:100%;
+  height:50px;
+}
+
+#menuToggle
+{
```

Prefer brace on same line for CSS (please change the others in this file too!)
```suggestion
#menuToggle {
```